aws/data_source_aws_ecs_service.go

```diff
@@ -0,0 +1,83 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsEcsService() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsEcsServiceRead,
+
+		Schema: map[string]*schema.Schema{
+			"service_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"cluster_arn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"desired_count": {
+				Type:     schema.TypeInt,
+				Optional: true,
```

Comment:
This attribute and the two below are not configurable (`Optional: true`) but rather read only and meant to just be set in the Terraform state (`Computed: true`). https://www.terraform.io/docs/extend/schemas/schema-behaviors.html